### PR TITLE
added image type validation in research notes

### LINKED ISSUE #1758: Add image extension validation when creating or updating a research note
plots2-bug1
I tried to upload an xlsx file in the file tag and it had no extension type validation, which resulted in a distorted **research note** as shown in the screenshot. 
Therefore,while creating or updating a research note, we should validate the image types (jpg,jpeg,png,etc.)

### PR DESCRIPTION
**Pull request by ViditChitkara**
fixes #1758

Make sure these boxes are checked before your pull request is ready to be reviewed and merged. Thanks!

* [X] all tests pass -- `rake test:all`
* [X] code is in uniquely-named feature branch, and has been rebased on top of latest master (especially if you've been asked to make additional changes)
* [X] pull request is descriptively named with #number reference back to original issue

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/wiki/contributing-to-public-lab-software

We have a loose schedule of reviewing and pulling in changes every Tuesday and Friday, and publishing changes on Fridays. Please alert developers on plots-dev@googlegroups.com when your request is ready or if you need assistance.

Thanks!

### CODE REVIEW

File: app/models/image.rb [hunk 1]
```diff
@@ -16,6 +16,7 @@ class Image < ActiveRecord::Base
   validates :uid, presence: :true
   validates :photo, presence: :true, unless: :remote_url_provided?
   do_not_validate_attachment_file_type :photo_file_name
+  validates_attachment_content_type :photo, :content_type => ['image/jpeg', 'image/png', 'image/jpg']
```
Comment (jywarren) [inline app/models/image.rb]: This looks great, but can we add `gif` too? Thanks!!

Comment (ViditChitkara) [reply app/models/image.rb]: @jywarren, sure I'll do.

**Comment by ViditChitkara** [comment]
@jywarren I have made the changes, please review!Thanks

**Comment by jywarren** [comment]
Great! Merging and thanks for your continued work!

**Comment by jywarren** [comment]
Hey, you're doing some great work, and we really appreciate it! Do you have any interest in helping to prepare some `first-timers-only` issues yourself, or respond to some other submissions to help others along? 

There's some guidance on this here, and we'd love to have your involvement: https://github.com/publiclab/plots2/projects/4

**Pull request merged**